test/integration_tests/with_db/utils/test_huggingface_hub_bucketfs_model_transfer_sp.py

```diff
@@ -1,93 +1,122 @@
 
+@contextlib.contextmanager
+def not_raises(exception):
+    try:
+        yield
+    except exception:
+        raise pytest.fail(f"Did raise {exception}")
 
-def test_download_with_model(bucketfs_location):
-    with tempfile.TemporaryDirectory() as folder:
-        test_setup = TestSetup(bucketfs_location)
-        base_model_factory: ModelFactoryProtocol = AutoModel
-        test_setup.downloader.download_from_huggingface_hub(
-            model_factory=base_model_factory
-        )
-        assert AutoModel.from_pretrained(
-            create_save_pretrained_model_path(
-                test_setup.downloader._tmpdir_name, test_setup.model_specification
-            )
-        )
-        del test_setup.downloader
 
+'''
+@pytest.fixture(scope="session")
+def bucketfs_location() -> bfs.path.PathLike:
+    import os
 
-def test_download_with_duplicate_model(bucketfs_location):
-    with tempfile.TemporaryDirectory() as folder:
-        test_setup = TestSetup(bucketfs_location)
-        base_model_factory: ModelFactoryProtocol = AutoModel
-        test_setup.downloader.download_from_huggingface_hub(
-            model_factory=base_model_factory
-        )
-        test_setup.downloader.download_from_huggingface_hub(
-            model_factory=base_model_factory
+    password = os.getenv("BUCKETFS_PASSWORD")
+    return bfs.path.build_path(
+        backend="onprem",
+        url="http://192.168.124.221:2580",
+        username="w",
+        password=password,
+        service_name="bfsdefault",
+        bucket_name="default",
+        verify=False,
+        path="",
+    )
+
+
+@pytest.fixture(scope="session")
+def db_conn():
+    return pyexasol.connect(dsn="192.168.124.221:8563", user="sys", password="exasol")
+
+
+@pytest.fixture(scope="session")
+def setup_database(db_conn):
+    return "TEST_TE_BFS_CONNECTION", ""
+
+
+def test_x1(db_conn, bucketfs_location):
+    result = db_conn.execute(
+        "SELECT {value}",
+        query_params={"value": "Hello"},
+    ).fetchone()
+    for f in bucketfs_location.iterdir():
+        print(f"{f}")
+    bfspath = bucketfs_location / "sub_dir" / "a.html"
+    with open("/home/chku/tmp/a.html", "br") as file:
+        bfspath.write(file)
+    print(f"uploaded to {bfspath}")
+
+
+@pytest.fixture(scope="session")
+def db_schema_name() -> str:
+    return "TE_ITEST"
+
+
+def test_x2(db_schema_name, db_conn):
+    print(f"{db_schema_name}")
+    query = cleandoc(
+        """
+        SELECT {db_schema_name}TE_MODEL_LOADER_UDF(
+          '{mspec.model_name}',
+          '{mspec.task_type}',
+          '{sub_dir}',
+          '{bucketfs_conn_name}'
         )
-        assert AutoModel.from_pretrained(
-            create_save_pretrained_model_path(
-                test_setup.downloader._tmpdir_name, test_setup.model_specification
-            )
+        """
+    )
+    result = db_conn.execute(query).fetchone()
+    print(f"{result}")
+'''
+
+
+def test_install_huggingface_model(
```

Comment:
this all seems kind of confusing to me. the HuggingfaceHubModelTransfere has not changed, why is the test changing so much? if you want a test for  the "install_huggingface_model", why not ad a file with a matching file name with that test in it? if you feel like those test would be redundant we can talk about removing this one, but as it is now, file name and actual test dont match.